cccombo.js

```diff
@@ -189,7 +189,8 @@ CccomboList.prototype = {
 function Cccombo(element) {
 	// Constructor
 	this.element = element;
-	this.input = this.element.querySelector('input:not([type="hidden"])');
+	this.input =
+		this.element.querySelector('.cccombo > input:not([type="hidden"])');
```

Comment:
Let's use [`:scope`](https://developer.mozilla.org/en-US/docs/Web/CSS/:scope), [it supported by IE](https://github.com/mdn/browser-compat-data/issues/846).

Reply:
Done.

Reply:
Sorry, my mistake, let's use `.cccombo >`. `:scope` isn't supported by IE.

Reply:
back done